threshold: 0.0
model_file: yolov8n-pose.pt
voxel_size: 0.06
subscribers:
  queue_size: 1
  exact_time: false
  slop: 1.0

  image_rgb: /kinect2/qhd/image_color_rect
  points: /kinect2/qhd/points

publishers:
  pose_detection:
    data: /butia_vision/pose 
    debug_image: /butia_vision/pose/image

  markers:
    topic: /butia_vision/pose/markers
    lifetime: 1